BUG-BOUNTY.md

```diff
@@ -1,10 +0,0 @@
-Serious about security
-======================
-
-Square recognizes the important contributions the security research community
-can make. We therefore encourage reporting security issues with the code
-contained in this repository.
-
-If you believe you have discovered a security vulnerability, please follow the
-guidelines at https://hackerone.com/square-open-source
```

Comment:
This contradicts the hackerone.com page which specifically tests for this file

https://hackerone.com/square-open-source

Reply:
Oooh, good catch. Lemme revert.